### PR TITLE
Formatting, along with field privacy and renaming Hatch to VelcroHatch.

### PR DESCRIPTION
**Pull request by TaeSeanDo**
Closes #16.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -10,5 +10,7 @@
     ".classpath": true,
     ".project": true
   },
-  "wpilib.online": true
+  "wpilib.online": true,
+"editor.detectIndentation": false,
+"editor.renderWhitespace": "boundary"
```
Comment (docprofsky) [inline .vscode/settings.json]: This is out of scope for this PR, but eh :man_shrugging: .

File: src/main/java/com/teamcautionrobotics/robot/Flashlight.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+package com.teamcautionrobotics.robot;
+
+import edu.wpi.first.wpilibj.*;
+
+public class Flashlight {
+   
+
+}
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/Flashlight.java]: This seems out of scope for this PR. Also, should there be an implementation for Flashlight?

File: src/main/java/com/teamcautionrobotics/robot/Robot.java [hunk 3]
```diff
@@ -25,131 +27,129 @@
...
+    double driveRightCommand;
+
+    Timer timer;
+
+    boolean deployButtonPressed = false;
+
+    private static final String kDefaultAuto = "Default";
+    private static final String kCustomAuto = "My Auto";
+    private String m_autoSelected;
+    private final SendableChooser<String> m_chooser = new SendableChooser<>();
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/Robot.java]: While we are doing the reformatting, these fields (lines 43 - 46) and all references to them (located in `autonomousInit` and `autonomousPeriodic`) can be deleted.

File: src/main/java/com/teamcautionrobotics/robot/Robot.java [hunk 4]
```diff
@@ -25,131 +27,129 @@
...
+     * between different autonomous modes using the dashboard. The sendable chooser
+     * code works with the Java SmartDashboard. If you prefer the LabVIEW Dashboard,
+     * remove all of the chooser code and uncomment the getString line to get the
+     * auto name from the text box below the Gyro
+     *
+     * <p>
+     * You can add additional auto modes by adding additional comparisons to the
+     * switch structure below with additional strings. If using the SendableChooser
+     * make sure to add them to the chooser code above as well.
+     */
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/Robot.java]: This whole comment can go.

File: src/main/java/com/teamcautionrobotics/robot/VelcroHatch.java [hunk 5]
```diff
@@ -20,7 +20,7 @@ public void rotate(double power) {
 
     public void deploy(boolean activate) {
         pusher.set(activate);
-        
+
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/VelcroHatch.java:23]: Remove extra line
```suggestion
```

File: src/main/java/com/teamcautionrobotics/robot/Robot.java [hunk 6]
```diff
@@ -12,7 +12,9 @@
 import com.teamcautionrobotics.misc2019.Gamepad.Axis;
 import com.teamcautionrobotics.misc2019.Gamepad.Button;
 
+
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/Robot.java]: Should this new line be here?

File: src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java [hunk 7]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java]: This new line probably does not belong here.
Also, surprise, a wild file appears.
```suggestion
```

File: src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    public void deploy(boolean extend) {
+        extender.set(extend);
+
+    }
+    public void pullback(boolean out) {
+        expander.set(out);
+    }
+
```
Comment (docprofsky) [inline src/main/java/com/teamcautionrobotics/robot/ExpanderHatch.java]: A wild newline appears.

**Review by docprofsky — CHANGES_REQUESTED** [review]
This is a very omnibus commit. Maybe also try to make pull requests smaller, with a narrower focus, and more atomic.

**Review by docprofsky — APPROVED** [review]

**Review by Ben1492 — APPROVED** [review]

**Pull request merged**